amm/contracts/pair/lib.rs

```diff
@@ -559,6 +566,23 @@ pub mod pair {
         }
     }
 
+    impl PSP22Metadata for PairContract {
+        #[ink(message)]
+        fn token_name(&self) -> Option<String> {
+            Some("Uniswap V2".to_string())
+        }
+
+        #[ink(message)]
+        fn token_symbol(&self) -> Option<String> {
+            Some("UNI-V2".to_string())
+        }
```

Comment:
Do we want to be that 1-1 with uniswap?

Reply:
I wanted to use the underlying tokens somehow but it'd have to by token addresses so not useful and the caller of these methods either already knows what are the underlying tokens or can query it if needed.

Reply:
I thought that maybe we want a more "common" name

Reply:
🙈 of course

Reply:
[Use Common-based names for PSP22Metadata](https://github.com/Cardinal-Cryptography/common-amm/pull/86/commits/4b018604aaabd98d2858f98bca774dd34cc6cb39)